pkg/cmd/pipeline/start.go

```diff
@@ -259,3 +294,20 @@ func parseParam(p []string) (map[string]v1alpha1.Param, error) {
 	}
 	return params, nil
 }
+
+func parseSvc(s []string) (map[string]v1alpha1.PipelineRunSpecServiceAccount, error) {
+	svcs := map[string]v1alpha1.PipelineRunSpecServiceAccount{}
+	for _, v := range s {
+		r := strings.Split(v, "=")
+		if len(r) != 2 || len(r[0]) == 0 {
+			errMsg := invalidSvc + v +
+				"\n Please pass service accounts as -r TaskName=ServiceAccount"
```

Comment:
nit: remove the space `\n Please` after new line.

How about `Please pass service accounts as -r TaskName=ServiceAccount` -> `Please pass service accounts as TaskName=ServiceAccount`?

Reply:
yeah `-r` doesn't exists